aws-ec2-manager-blueprint-inputs.yaml

```diff
@@ -136,22 +153,35 @@ ec2_region_name: ''
 #  - {'source_path': '/cfy/cloudify/plugins/script-plugin/plugin.yaml', 'destination_path': '/script-plugin/1.3rc1/plugin.yaml'}
 #  - {'source_path': '/cfy/cloudify/plugins/diamond-plugin/plugin.yaml', 'destination_path': '/diamond-plugin/1.3rc1/plugin.yaml'}
+#  - {'source_path': '/cfy/cloudify/plugins/openstack-plugin/plugin.yaml', 'destination_path': '/openstack-plugin/1.3rc1/plugin.yaml'}
 #  - {'source_path': '/cfy/cloudify/plugins/aws-plugin/plugin.yaml', 'destination_path': '/aws-plugin/1.3rc1/plugin.yaml'}
+#  - {'source_path': '/cfy/cloudify/plugins/tosca-vcloud-plugin/plugin.yaml', 'destination_path': '/tosca-vcloud-plugin/1.3rc1/plugin.yaml'}
 #  - {'source_path': '/cfy/cloudify/types/types.yaml', 'destination_path': '/cloudify/3.3rc1/types.yaml'}
 #plugin_resources:
 #################
 # Diamond plugin
 #################
-#  - 'http://gigaspaces-repository-eu.s3.amazonaws.com/org/cloudify3/3.3.0/rc1-RELEASE/cloudify_diamond_plugin-1.3a7-py26-none-linux_x86_64-centos-Final.tar.gz'
-#  - 'http://gigaspaces-repository-eu.s3.amazonaws.com/org/cloudify3/3.3.0/rc1-RELEASE/cloudify_diamond_plugin-1.3a7-py27-none-linux_x86_64-Ubuntu-precise.tar.gz'
-#  - 'http://gigaspaces-repository-eu.s3.amazonaws.com/org/cloudify3/3.3.0/rc1-RELEASE/cloudify_diamond_plugin-1.3a7-py27-none-linux_x86_64-Ubuntu-trusty.tar.gz'
-#  - 'http://gigaspaces-repository-eu.s3.amazonaws.com/org/cloudify3/3.3.0/rc1-RELEASE/cloudify_diamond_plugin-1.3a7-py27-none-win32-none-none.tar.gz'
+#  - 'http://gigaspaces-repository-eu.s3.amazonaws.com/org/cloudify3/3.3.0/rc1-RELEASE/cloudify_diamond_plugin-1.3rc1-py26-none-linux_x86_64-centos-Final.tar.gz'
+#  - 'http://gigaspaces-repository-eu.s3.amazonaws.com/org/cloudify3/3.3.0/rc1-RELEASE/cloudify_diamond_plugin-1.3rc1-py27-none-linux_x86_64-Ubuntu-precise.tar.gz'
+#  - 'http://gigaspaces-repository-eu.s3.amazonaws.com/org/cloudify3/3.3.0/rc1-RELEASE/cloudify_diamond_plugin-1.3rc1-py27-none-linux_x86_64-Ubuntu-trusty.tar.gz'
+#  - 'http://gigaspaces-repository-eu.s3.amazonaws.com/org/cloudify3/3.3.0/rc1-RELEASE/cloudify_diamond_plugin-1.3rc1-py27-none-win32-none-none.tar.gz'
 #################
 ## Fabric plugin
 #################
-#  - 'http://gigaspaces-repository-eu.s3.amazonaws.com/org/cloudify3/3.3.0/rc1-RELEASE/cloudify_fabric_plugin-1.3a7-py27-none-linux_x86_64-centos-Core.tar.gz'
+#  - 'http://gigaspaces-repository-eu.s3.amazonaws.com/org/cloudify3/3.3.0/rc1-RELEASE/cloudify_fabric_plugin-1.3rc1-py27-none-linux_x86_64-centos-Core.tar.gz'
 ##############
 ## Aws plugin
 ##############
-#  - 'http://gigaspaces-repository-eu.s3.amazonaws.com/org/cloudify3/3.3.0/rc1-RELEASE/cloudify_aws_plugin-1.3a7-py27-none-linux_x86_64-centos-Core.tar.gz'
-
+#  - 'http://gigaspaces-repository-eu.s3.amazonaws.com/org/cloudify3/3.3.0/rc1-RELEASE/cloudify_aws_plugin-1.3rc1-py27-none-linux_x86_64-centos-Core.tar.gz'
+####################
+## Openstack plugin
```

Comment:
For some reason, they decided to keep those commented out in the inputs, but the defaults in the blueprints itself do contain only the relevant ones...